Formula/apbspdb2pqr.rb

```diff
@@ -0,0 +1,56 @@
+class Apbspdb2pqr < Formula
+  include Language::Python::Virtualenv
+  # cite Jurrus_2018: "https://doi.org/10.1002/pro.3280"
+  # cite Baker_2001: "https://doi.org/10.1073/pnas.181342398"
+  desc "Electrostatic and solvation properties for complex molecules"
+  homepage "https://www.poissonboltzmann.org/"
+  # pull from git tag to get submodules
+  url "https://github.com/Electrostatics/apbs-pdb2pqr.git",
+      :tag      => "apbs-1.5",
+      :revision => "aa353941cfadc09ccd113075d261a427864c2979"
+  head "https://github.com/Electrostatics/apbs-pdb2pqr.git"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
+  depends_on "eigen"
+  depends_on "python"
+  depends_on "python@2"
+  depends_on "swig"
+
+  def install
+    # APBS part
+    # Disable finite element support because its LD_LIBRARY_PATH is complicated.
+    args = std_cmake_args + %w[
+      -DCMAKE_POLICY_DEFAULT_CMP0046=OLD
+      -DCMAKE_POLICY_DEFAULT_CMP0078=OLD
+      -DCMAKE_POLICY_DEFAULT_CMP0086=OLD
+      -DENABLE_GEOFLOW=ON
+      -DENABLE_FETK=OFF
+      -DENABLE_PBAM=ON
+      -DGET_MSMS=ON
+      -DGET_NanoShaper=ON
+      -DENABLE_PYTHON=ON
+      -DBUILD_DOC=OFF
+      -DCMAKE_BUILD_TYPE=Release
+    ]
+    args << "-DBUILD_SHARED_LIBS=OFF" if OS.linux?
+    system "cmake", "apbs", *args
+    ENV.deparallelize { system "make", "install" }
+
+    # PDB2PQR part
+    cd "pdb2pqr" do
+      system "python", "scons/scons.py", "PREFIX=#{prefix}/pdb2pqr", "APBS=/usr/local/bin/apbs", "BUILD_PDB2PKA=False"
```

Comment:
Is the `/usr/local/bin/apbs` correct here?

Reply:
Thank you for your suggestion. I have replaced with `APBS=#{bin}/apbs`.